src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -59,9 +59,9 @@ type Fetcher interface {
 }
 
 // StartPolling starts the process of polling for new configuration files.
-func StartPolling() {
+func (cfg *Config) StartPolling() {
 	// Force detour to whitelist chained domain
-	u, err := url.Parse(chainedCloudConfigURL)
+	u, err := url.Parse(cfg.CloudConfig)
```

Comment:
We should really be using the configured server here rather than the default -- made this a method of the struct accordingly.